### PR TITLE
Change cli logs folder location

### LINKED ISSUE #473: the logs folder is not located in the right place [1]
logs folder should be somewhere in the cslang root folder maybe next to the bin folder

now it created the logs folder under content...

### PR DESCRIPTION
**Pull request by maayan-avraham**
fixes #473 
Signed-off-by: Maayan Avraham maayan.avraham@hpe.com

### CODE REVIEW

**Comment by maayan-avraham** [comment]
@Bonczidai please review.

**Comment by Bonczidai** [comment]
looks good - but note we have this situation:

Ubuntu 14.04 with this folder hierarchy: `/root/path_test/a/b/c/cslang/bin` - you are in `path_test` and run the cmd `a/b/c/cslang/bin/cslang`

`logs` is created in `root`, should be beside `bin`

@maayan-avraham , can you please look into it?

File: cloudslang-cli/src/main/resources/log4j.properties [hunk 1]
```diff
@@ -4,7 +4,7 @@ def.files.backup.count=10
 log4j.rootLogger=INFO, A1
 
 log4j.appender.A1=org.apache.log4j.RollingFileAppender
-log4j.appender.A1.File=logs/execution.log
+log4j.appender.A1.File=${user.dir}/target/cslang/logs/execution.log
```
Comment (Bonczidai) [inline cloudslang-cli/src/main/resources/log4j.properties]: seems to me that `${user.dir}` returns the same as `.` which does not solve [this issue](https://github.com/CloudSlang/cloud-slang/pull/482#issuecomment-157315259)

please investigate further

Comment (Bonczidai) [inline cloudslang-cli/src/main/resources/log4j.properties]: e.g. `getClass().getResource("")` returns a STR tat contains path to `lib` folder - you may be able to use this (please check the other possibilities, there may be better ones)

**Pull request merged**